test/integration/kafka_sarama/kafka_test.go

```diff
@@ -30,7 +33,11 @@ func TestSendEvent(t *testing.T) {
 		clienttest.SendReceive(t, func() interface{} {
 			return protocolFactory(t)
 		}, eventIn, func(e event.Event) {
-			test.AssertEventEquals(t, eventIn, test.ConvertEventExtensionsToString(t, e))
+			got := test.ConvertEventExtensionsToString(t, e)
+			eventIn.SetExtension(KAFKA_OFFSET, got.Extensions()[KAFKA_OFFSET])
+			eventIn.SetExtension(KAFKA_PARTITION, got.Extensions()[KAFKA_PARTITION])
+			eventIn.SetExtension(KAFKA_TOPIC, got.Extensions()[KAFKA_TOPIC])
```

Comment:
There is another way we can remove the `KAFKA_OFFSET `, `KAFKA_PARTITION`, and `KAFKA_TOPIC ` from `got`. Is this way better?

Reply:
I'm curious, how do we remove extension attributes from a CE? I didn't see that option in here: https://github.com/cloudevents/sdk-go/blob/main/v2/event/event_writer.go

I'm ok with copying things from `got` into `eventIn` so that the Assert works (or removing them from `got` if that's possible), but it seems to me that there should be a check of the values in `got` to make sure they're not blank - even if we need to do those manually instead via a compare with `eventIn`.

Reply:
@duglin Thanks for your advice! I have added the empty check for the event extensions.